src/backend/controllers/bookingTypeControllers.ts

```diff
@@ -25,28 +29,43 @@ export async function addBookingType({
     : null;
 }
 
-export async function getBookingTypes(): Promise<Array<ApiBookingType> | null> {
+export async function getBookingTypes(): Promise<Array<ApiBookingTypeWithColor> | null> {
   const allBookingTypes = await model.getAllBookingTypes();
   return allBookingTypes !== null
-    ? allBookingTypes.map((b) => ({
-        uuid: b.uuid,
-        name: b.name,
-        rules: b.rules,
-        exceptions: b.exceptions,
-        additionalInformation: b.additionalInformation,
-      }))
+    ? allBookingTypes
+        .sort((a, b) => new Date(a.created).getTime() - new Date(b.created).getTime())
+        .map((b, index) => ({
+          uuid: b.uuid,
+          name: b.name,
+          rules: b.rules,
+          exceptions: b.exceptions,
+          additionalInformation: b.additionalInformation,
+          color: BookingTypeColors[index % Object.keys(BookingTypeColors).length],
```

Comment:
There is no big reason for that. I was thinking maybe later (if we are going to store it in DB) this could be described in word rather than the rgba code.

Reply:
Then I think it should be enough to change that into an array, or maybe an object with the numbers as keys if you want to keep the possibility to easily use different names later. You will do the change right?